lib/runtime/allocator_test.go

```diff
@@ -264,8 +264,25 @@ func TestAllocator(t *testing.T) {
 		const size = 1 << 16
 		testobj := make([]byte, size)
 
-		memmock.EXPECT().Data().Return(testobj).AnyTimes()
-		memmock.EXPECT().Length().DoAndReturn(func() uint32 {
+		memmock.EXPECT().WriteByte(gomock.Any(), gomock.Any()).DoAndReturn(func(offset uint32, v byte) bool {
+			testobj[offset] = v
+			return true
+		}).AnyTimes()
```

Comment:
Given the @axaysagathiya PR to replace the all `AnyTimes()` occurrences, I think would be nice to change here as well to the right number using `Times(n)`

Reply:
given the nature of the tests and the fact that it references `allTests`, I don't think it makes much sense in this case to arbitrarily ensure that the mocked methods are called at `n` times.  This will make it pretty difficult if we ever decided to add something to `allTests`.